build.psm1

```diff
@@ -1844,6 +1844,9 @@ function New-MSIPackage
     }
     else
     {
+        $WiXHeatLog   | Out-String | Write-Verbose
+        $WiXCandleLog | Out-String | Write-Verbose
+        $WiXLightLog  | Out-String | Write-Verbose
```

Comment:
I think you probably need `Write-Verbose -Verbose`, otherwise, the verbose message won't be written out unless `New-MSIPackage` is called with `-Verbose`.

Reply:
Ok. I fixed it. But why is the build script no being called using the `-Verbose` option from the top level? As long as advanced functions are used, the `$VerbosePreference would be propagated through. If you always force verbose output, what is then the difference to `Write-Host`? I am thinking about [this](http://www.jsnover.com/blog/2013/12/07/write-host-considered-harmful/) blog post from Jeffrey.

Reply:
`-Verbose` is not specified when calling [`Invoke-AppveyorFinish`](https://github.com/PowerShell/PowerShell/blob/master/appveyor.yml#L28) and also not specified when calling [`Start-PSPackage`](https://github.com/PowerShell/PowerShell/blob/master/tools/appveyor.psm1#L434), as well as when calling [`New-MSIPackage`](https://github.com/PowerShell/PowerShell/blob/master/tools/packaging/packaging.psm1#L167). Maybe it should be specified at the very top call.

But for this change, we'd better add `-Verbose` anyway because even if `-Verbose` is not specified when calling `New-MSIPackage`, it's still desired to write out the logs when no MSI was produced.